### DISCUSSION TITLE
Join Community Button

### DISCUSSION THREAD

**Discussion by warengonzaga** [web3phl/directory#59] [Brainstorm]
Hey ppl, it's nice to have a join community button on our directory site. Can you propose @jnebab a button design for this one? We need to push this update to v1.2.0 at #35

**Comment by jnebab** [comment]
we can initially do same as the Submit Project button for now. Make them side by side in the header.

**Reply by warengonzaga** [reply]
Yeah agree on that, can you take this? Only for Facebook and Discord join buttons
